client/web/compose/src/components/Chart/ChartTranslator.vue

```diff
@@ -45,6 +46,11 @@ export default {
       type: Boolean,
       default: () => false,
     },
+
+    tooltip: {
```

Comment:
since you do a v-bind to the c-translator-button, you don't need to define the prop